mxnet/inference/docker/1.7.0/py3/cu101/Dockerfile.gpu

```diff
@@ -93,6 +93,8 @@ RUN ${PIP} --no-cache-dir install --upgrade \
 WORKDIR /
 
 RUN ${PIP} install --no-cache-dir \
+   "awscli<2" \
```

Comment:
likely it's a good idea to restrict the major versions of both of these packages to avoid conflicts or breaking changes

Reply:
awscli > 2 has breaking changes, so <2 and boto3 is always latest we install. It is as per all other latest framework version dockerfiles as well.

Reply:
Given the current state of boto3/awscli, we should be fine on potential botocore conflicts. We can potentially use a package manager in the future to ensure we don't run into dependency conflicts